src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -312,6 +340,37 @@ private Response invokeCallbackEndpoint(
             .post("/callback/attach_case?ignore-warning=true");
     }
 
+    private Response invokeCallbackEndpointWithSearchCaseRefAndAttachToCaseRef(
+        CaseDetails searchCaseRefCaseDetails,
+        CaseDetails exceptionRecord
+    ) {
+        Map<String, Object> exceptionRecordData = new HashMap<>(exceptionRecord.getData());
+        exceptionRecordData.put("searchCaseReference", String.valueOf(searchCaseRefCaseDetails.getId()));
+
+        CaseDetails exceptionRecordWithSearchFields =
+            exceptionRecord.toBuilder().data(exceptionRecordData).build();
+
+        CallbackRequest callbackRequest = CallbackRequest
+            .builder()
+            .eventId("attachToExistingCase")
+            .caseDetails(exceptionRecordWithSearchFields)
+            .build();
+
+        CcdAuthenticator ccdAuthenticator = ccdAuthenticatorFactory.createForJurisdiction("BULKSCAN");
+
+        return RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(testUrl)
+            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator Functional test")
+            .header(AUTHORIZATION, ccdAuthenticator.getUserToken())
+            .header(CcdCallbackController.USER_ID, ccdAuthenticator.getUserDetails().getId())
+            .body(callbackRequest)
+            .when()
+            .post("/callback/attach_case?ignore-warning=true");
```

Comment:
warning can be false here ? No warnings list is populated so ignore warning = true would not be the right scenario